src/jti_content.js

```diff
@@ -20,6 +20,8 @@ if (navigator.userAgent.indexOf('Firefox') !== -1 || navigator.userAgent.indexOf
 // Handle <TI> tag selection.
 $(document).on('click', '#description', function () {
     var text = $(this).val(),
+        ctrlDown = false,
+        backtickKey = 192,

```

Comment:
We're declaring the backtickKey value here, but we're using the key value ("17") directly for the crtl key. I think to be consistent we should either declare both `backtickKey = 192` and `ctrlKey = 17` and use the variables, or neither and use the values directly. 
I would lean towards declaring the variables.